list_port_forwards/display_port_forwards.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "__(.)= __(.)=    =(.)__ =(.)__"
+echo "\___)  \___)      (___/  (___/"
+
+echo "---- Port forwards in use ----"
+
+jqquery='.items[] | { name:.metadata.annotations."tilt.dev/resource", p:.spec.forwards[] } | "\(.name):\(.p.containerPort) -> \(.p.host):\(.p.localPort)"'
+
+startquote='s/^"//'
```

Comment:
i think `jq -r` will omit the quotes for you, so you don't need to do it manually

Reply:
Changed. Thank you!